deployment/helm/synopsys-alert/values.yaml

```diff
@@ -37,8 +35,6 @@ cfssl:
   resources:
     limits:
       memory: "640Mi"
-    requests:
-      memory: "640Mi"
```

Comment:
Why are we removing the memory here and above?

Reply:
@msenmurugan you will be able to answer this better.

Reply:
@psantos1113 I have already specified in the description of the PR

If you specify limits alone on kubernetes container resources, then internally Kubernetes will set requests as same as limits and hence removed requests from resources in order to remove the duplicates

If customer want to keep different requests and limits, they can provide those values as part of alert.resources.requests.memory flag